apps/chat/src/view/ChatItem.tsx

```diff
@@ -111,14 +113,15 @@ export const ChatItem = ({
             <Markdown
               children={sanitizedAnswer}
               options={{
-                forceBlock: true,
+                forceInline: true,
                 overrides: {
                   SummaryCitation: {
                     component: SummaryCitation
                   }
                 }
               }}
             />
+            <span>{isSummarizing && <VuiSpinner size="xs" />}</span>
```

Comment:
Can we capture the span within the condition, and also use a space character instead of adding margin to the spinner with CSS? The space will be handled better by the browser. With the current implementation you can get into a state like this:

![image](https://github.com/vectara/create-ui/assets/1238659/d3131448-874d-43a4-9118-b6e71182b819)

Suggestion:

```tsx
{isSummarizing && <span>{" "}<VuiSpinner size="xs" /></span>}
```